www/widgets/DemoExampleMaker.js

```diff
@@ -86,6 +86,10 @@ class DemoExampleMaker extends Widget {
           display: flex;
           justify-content: space-between;
         }
+        #dem-remove-step:hover {
+          background-color: rgb(252, 97, 86);
+          border-color: rgb(252, 97, 86);
+        }
```

Comment:
let's change the selector here to avoid using `id`, this is probably a specific enough id that it shouldn't cause issues w/future widgets, but just to be extra safe i've been using this overly cautious/specific-to-this-widget approach:

```css
let's remove this `style="float: right"` from here && add it the CSS class u created:

```css
.demo-example-maker  button[name="dem-remove-step"]:hover  {
  background-color: rgb(252, 97, 86);
  border-color: rgb(252, 97, 86);
}
```